spec/services/support_interface/start_of_cycle_notifier_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe SupportInterface::StartOfCycleNotifier do
+  describe '#call' do
+    it 'does nothing when called on a day that is not the start of the cycle for the service' do
+      Timecop.freeze(2021, 9, 7) do
```

Comment:
I wonder if we could make the time relative, something like this:`Timecop.freeze(CycleTimetable.apply_1_deadline + 1.day)`, same below?